grails-app/services/org/broadinstitute/orsp/StorageProviderService.groovy

```diff
@@ -163,7 +163,7 @@ class StorageProviderService implements Status {
      * @param files The multipart files
      * @return
      */
-    StorageDocument saveMultipartFile(String displayName, String userName, String issueKey, String type, MultipartFile file) {
+    StorageDocument saveMultipartFile(String displayName, String userName, String issueKey, String type, MultipartFile file, String consentKey) {
```

Comment:
This is a mistake, please revert. `issueKey` is where you should be storing the key of the consent's project key. This will end up breaking all existing storage documents for consent groups where the project key of the consent group is already stored in `issueKey`.